_data/team.yml

```diff
@@ -46,4 +46,14 @@ people:
     member_card_title: Thomas Nguyen
     member_card_text: Thomas graduated from Virginia Tech with a bachelor’s degree in Computer Science. His knowledge of system and microservices architecture, development, and sustainment on containers and integration with Amazon Web Services (AWS) nurtures optimize systems. Thomas is experienced in developing features on frontend and backend, administering/securing systems with vulnerability scans and updates, and migrating both personal health information (PHI) and non-PHI data. Thomas understands delivering quality releases with proper version control integrating with multiple dependencies and APIs. He applies his previous knowledge from maintaining Department of Defense (DoD) subnautical systems, developing and maintaining Department of Veteran Affairs (DoVA) health systems, and modernizing Centers for Disease Control and Prevention (CDC) vaccination system towards contemporary systems and services to empower users to solve bigger problems. 
 
+  - name: Chief Strategy Officer
+    pic: austin
+    positions:
+      - ""
```

Comment:
why is this `positions: [ "" ]`, seems like 
```
positions: []
````
makes more sense

Reply:
> Do we need this `webp/team/austin.webp` file?

This is the compressed file that is loaded on compatible browsers to optimize image load.



> why is this `positions: [ "" ]`, seems like
> 
> ```
> positions: []
> ```
> 
> makes more sense

this was debt, I cleaned it up to just include a single position for now. 


![image](https://user-images.githubusercontent.com/47189388/166834251-c50c89a9-9ca3-4ea0-99b3-a14c89dd639b.png)